src/model_signing/manifest.py

```diff
@@ -245,16 +246,156 @@ def key(self) -> ManifestKey:
         return Shard(self._path, self._start, self._end)
 
 
+class SerializationType(metaclass=abc.ABCMeta):
```

Comment:
its subclasses are private, does this need to be?

I see it's passed into the Manifest initializer, but are we expecting others to do so?

Reply:
I still have a TODO item to go over all the files and add/remove `_` to make the APIs properly public private. For now I  know for sure that the subclasses don't need to be exposed, but the rest not sure yet